stregsystem/management/commands/importmobilepaypayments.py

```diff
@@ -16,12 +17,14 @@
 class Command(BaseCommand):
     help = 'Imports the latest payments from MobilePay'
 
-    api_endpoint = 'https://api.mobilepay.dk'
+    api_endpoint = 'https://api.vipps.no'
     # Saves secret tokens to the file "tokens.json" right next to this file.
     # Important to use a separate file since the tokens can change and is thus not suitable for django settings.
     tokens_file = (Path(__file__).parent / 'tokens.json').as_posix()
     tokens_file_backup = (Path(__file__).parent / 'tokens.json.bak').as_posix()
     tokens = None
+    manual_cutoff_date = date(2024, 4, 9)
```

Comment:
```suggestion
    # Cutoff for when this iteration of the Mobilepay-API (Vipps) is deployed
    manual_cutoff_date = date(2024, 4, 9)
```

For future reference